### ISSUE TITLE
Can't Install hoppscotch on Edge

### ISSUE THREAD

**Issue by teofilosalgado** [hoppscotch/hoppscotch#1931]
**Describe the bug**
I am not able to install hoppscotch through the install button. Its giving the following error in Microsoft's Edge console:
![image](https://user-images.githubusercontent.com/7178504/140105670-e01afd72-03ca-405e-9da4-b0dc745a63f1.png)
Transcript:
```
TypeError: t.showInstallPrompt is not a function
    at click (3ef226b.modern.js:1)
    at ee (f8e0c2f.modern.js:2)
    at HTMLButtonElement.n (f8e0c2f.modern.js:2)
    at HTMLButtonElement.Mr.c._wrapper (f8e0c2f.modern.js:2)
```

**To Reproduce**
1. Go to https://hoppscotch.io/
2. Click on Install

**Expected behavior**
The application should install accordingly.

**Screenshots**
![image](https://user-images.githubusercontent.com/7178504/140106539-23f26ea0-c53d-44fd-ab19-6761c37eb977.png)
![image](https://user-images.githubusercontent.com/7178504/140105670-e01afd72-03ca-405e-9da4-b0dc745a63f1.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 21H1
 - Browser Microsoft Edge
 - Version 95.0.1020.40 (64 bits)

**Comment by AndrewBastin** [comment]
Thank you for reporting @teofilosalgado!

Will look into this ASAP

**Comment by AndrewBastin** [comment]
Fixed this in #1932 !

Should be live in a couple of minutes, please report if it still persists after that @teofilosalgado

**Comment by teofilosalgado** [comment]
It's working like a charm! Thank you very much @AndrewBastin! _Fastest fix in the west!_ 🤠
![image](https://user-images.githubusercontent.com/7178504/140173483-2988e4cb-8fa2-47f5-8c6c-4a30f964e811.png)
